### ISSUE TITLE
[DataCap Application] <AWS> - <Public Dataset-ACIC/CZ CELLxGENE/GeoNet>

### ISSUE THREAD

**Issue by Friedawoo** [Carohere/Caro-Allocator#23]
### Data Owner Name

ALLEN INSTITUTE/Chan Zuckerberg Initiative/GeoNet

### Data Owner Country/Region

United States

### Data Owner Industry

Life Science / Healthcare

### Website

https://www.allencell.org/ https://chanzuckerberg.com/ https://www.geonet.org.nz/

### Social Media Handle

Frieda

### Social Media Type

Slack

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

2.5PiB

### Expected size of single dataset (one copy)

194.9TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

200TiB

### On-chain address for first allocation

f1higevr5a34m4dqumr4flwd7wfstnby2t7qfawpi

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
The mission of Allen Institute is to understand and predict cell behaviors.
CZ CELLxGENE Discover (cellxgene.cziscience.com) is a free-to-use platform for the exploration, analysis, and retrieval of single-cell data. CZ CELLxGENE Discover hosts the largest aggregation of standardized single-cell data from the major human and mouse tissues, with modalities that include gene expression, chromatin accessibility, DNA methylation, and spatial transcriptomics.
GeoNet is the result of a partnership between Natural Hazards Commission Toka Tū Ake, GNS Science, and Land Information New Zealand (LINZ). The GeoNet project was established in 2001 to build and operate a modern geological hazard monitoring system in New Zealand. It comprises a network of geophysical instruments, automated software applications and skilled staff to detect, analyse and respond to earthquakes, volcanic activity, large landslides, tsunami and the slow deformation that precedes large earthquakes.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
This bucket contains multiple datasets (as Quilt packages) created by the Allen Institute for Cell Science.
Human and mouse RNA single-cell data
GeoNet provides geological hazard information for Aotearoa New Zealand. This dataset contains data and products recorded by the GeoNet sensor network.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

Hong Kong

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
We are a technical team with the ability to prepare data. We use lotus to prepare data.
```

### Has this dataset been stored on the Filecoin network before? If so, please explain and make the case why you would like to store this dataset again to the network. Provide details on preparation and/or SP distribution.

```text
No. We have confirmed.
```

### Please share a sample of the data

```text
aws s3 ls --no-sign-request s3://allencell/
aws s3 ls --no-sign-request s3://cellxgene-census-public-us-west-2/cell-census/
aws s3 ls --no-sign-request s3://geonet-open-data/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Monthly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, South America, Europe

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Big Data Exchange

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f03241837, Zhejiang
f03216485, Sichuan
f03100004, Guangdong
f03214920, HongKong
f03231666, Sichuan
```

### How do you plan to make deals to your storage providers

Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by Carohere** [comment]
Hello @Friedawoo , please answer these questions as a survey :

1. Can you give me your email via slack so that I can contact you in another way.
2. How did you hear about the FIL+ project?
3. What kind of data do you plan to store?
4. Have you previously received DataCap and have you finished with the whole usage? If so, please share the link to your previous request/ github issue.
5. Have you applied for DataCap from another allocator, and has it been accepted or rejected? If yes, please share the link to the previous application

**Comment by Carohere** [comment]
@Friedawoo Got your slack dm and sent an email to you. If you have any questions, you can call me via slack or email.
As a new joiner. do you know how to find sps with good retrieval rate?

**Comment by Carohere** [comment]
@Friedawoo You should keep good retrieval and I will check it any time. If you need to add or change sps, please give me the list in your application.
Alright. When I'm ready, I will allocate you the first round of datacap as a chance.

**Comment by Friedawoo** [comment]
@Carohere Thanks Caro.

**Comment by Carohere** [comment]
checker:manualTrigger

**Comment by Friedawoo** [comment]
Hi Caro @Carohere , we have used 86% datacap till today. Can you sign for the next round for us in order to go ahead to the next step?

**Comment by Carohere** [comment]
@Friedawoo Sps are all in the list you provided. All sps support retrieval. Good job!
![image](https://github.com/user-attachments/assets/bdc552aa-c26a-48e2-ba29-05c9bfddf17f)
Based on your report, we can support you and allocate datacap for another round.

**Comment by Friedawoo** [comment]
@Carohere We come to give the sp list about the partners we are cooperating with  now.
f03241837, Zhejiang
f02851143, Luzhou
f01431043, Hongkong
f03100004, Guangdong
f03216485, Sichuan
f03231666, Sichuan
f01660795, Guangdong

**Comment by Carohere** [comment]
checker:manualTrigger

**Comment by Carohere** [comment]
@Friedawoo All sps are in the latest sp list you provided.
![image](https://github.com/user-attachments/assets/2e23347c-8794-459a-be34-f778d4ef6cce)
One sps's successful retrieval rate is low. You should let them resolve the problem or you stop to send data to them. Can you do that?

**Comment by Friedawoo** [comment]
@Carohere Yes. We have stopped to work with this sp with low retrieval. When we get into next round, we will invite new sps to finish the storage.

**Comment by Carohere** [comment]
@Friedawoo Retrieval is the most important standard in the process. Go for pay more attention please.
 You can get another round of datacap as your plan.

**Comment by Carohere** [comment]
checker:manualTrigger

**Comment by Friedawoo** [comment]
@Carohere We invite a new sp f03099981, HongKong. They support retrieval and we will keep improving our work.

**Comment by Carohere** [comment]
checker:manualTrigger

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by Carohere** [comment]
checker:manualTrigger

**Comment by Friedawoo** [comment]
@Carohere We found two sps to finish our storage.
f0671982, HongKong
f03251993, Ningbo
But one sp need resolve their retrieval, we are communicating about it and stop sending data to this sp now.

**Comment by Carohere** [comment]
Total datacap reached and close the application.
